yomi/README.md

```diff
@@ -0,0 +1,8 @@
+# Yomi Server
+
+[Yomi](https://senseis.xmp.net/?Yomi) means reading in Japanese Go Jargon, and
+in the context of Minigo, is the name of a kubrnetes workload designed to play
+games against various engines (mostly itself).
```

Comment:
put in a TODO about adding a sentence or two of the architecture overview here.

Reply:
Done